crates/astria-cli/src/cli/sequencer.rs

```diff
@@ -96,16 +101,29 @@ pub struct BasicAccountArgs {
     )]
     pub(crate) sequencer_url: String,
     /// The address of the Sequencer account
-    pub(crate) address: SequencerAddressArg,
+    pub(crate) address: Address,
+}
+
+#[derive(Args, Debug)]
+pub struct Bech32mAddressArgs {
+    /// The hex formatted byte part of the bech32m address
+    #[arg(long)]
+    pub(crate) bytes: String,
+    /// The human readable prefix (Hrp) of the bech32m adress
+    #[arg(long, default_value = "astria")]
+    pub(crate) prefix: String,
 }
 
 #[derive(Args, Debug)]
 pub struct TransferArgs {
     // The address of the Sequencer account to send amount to
-    pub(crate) to_address: SequencerAddressArg,
+    pub(crate) to_address: Address,
     // The amount being sent
     #[arg(long)]
     pub(crate) amount: u128,
+    /// The bech32m prefix that will be used for constructing addresses using the private key
+    #[arg(long)]
+    pub(crate) prefix: String,
```

Comment:
is this just for logging? Doesn't seem necessary?

Reply:
It's needed to construct an address and get the latest nonce - although if you follow the calls in `sequencer_client::SequencerClientExt`, then ultimately it only uses the bytes contained in the address. All of this looks pretty bad.

I am updating the queries in sequencer to parse the addresses from the provided paths.

Reply:
Turns out the queries didn't need updating - just `astria-sequencer-client` would have been broken because sequencer was already updated.

I have updated all commands that take `--prefix` to use `"astria"` as the default.